tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPropertyNameMatchingTest.java

```diff
@@ -43,7 +43,7 @@
  * The following environment variables must be set prior to running this test:
  * "my_int_property" with the value of "45"
  * "MY_BOOLEAN_PROPERTY" with the value of "true"
- * "my_string_property" with the value of "haha"
+ * "MY.STRING_PROPERTY" with the value of "haha"
```

Comment:
This change is not valid. The environment variable can not contain dots.

What is the intent of your PR?

Reply:
@jmesnil In Windows, it cannot differentiate the case. my.string.property is treated as the same as MY.STRING.PROPERTY. The test will not pass when running on Windows. Maybe we should ignore this test. Thoughts?